builtin/providers/clc/resource_clc_server.go

```diff
@@ -44,15 +44,14 @@ func resourceCLCServer() *schema.Resource {
 				Type:     schema.TypeInt,
 				Required: true,
 			},
+			// optional
 			"password": &schema.Schema{
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
```

Comment:
As the code now makes this optional, we need to change the 

`func resourceCLCServerCreate(d *schema.ResourceData, meta interface{}) error {` func so that we only pass a password if it is supplied. Right now, the Create will supply a null value